axum/src/docs/routing/route_layer.md

```diff
@@ -26,10 +29,11 @@ use tower_http::validate_request::ValidateRequestHeaderLayer;
 
 let app = Router::new()
     .route("/foo", get(|| async {}))
+    .fallback(|| async { "fallback" })
     .route_layer(ValidateRequestHeaderLayer::bearer("password"));
 
 // `GET /foo` with a valid token will receive `200 OK`
 // `GET /foo` with a invalid token will receive `401 Unauthorized`
-// `GET /not-found` with a invalid token will receive `404 Not Found`
+// `GET /not-found` with a invalid token will receive `200 OK` (fallback)
```

Comment:
I don't think this example benefits from having an explicit fallback handler. Why did you add that? Maybe just add `(default fallback handler)` after the `404 Not Found`?